src/Autosuggest.js

```diff
@@ -140,6 +145,18 @@ export default class Autosuggest extends Component {
     }
 
     this.setState(newState);
+
+    this.onSuggestionUnfocused();
+    if(suggestion != null) {
```

Comment:
I prefer we use strict equality to keep things as explicit as possible. In this scenario, I believe `suggestion` can be `undefined` because that's what `getSuggestion(sectionIndex, suggestionIndex)` will return when `suggestionIndex === null`. The convention that I've been using to identify whether a suggestion is focused is to check whether `suggestionIndex === null`, so let's keep with that. We could have a `isSuggestionFocused()` function, if that make sense to add it.